### DISCUSSION TITLE
PostCSS - SCSS

### DISCUSSION THREAD

**Discussion by bulentevren** [jpanther/congo#488] [Q&A] [answered]
Does congo has builtin feature to extract unused css from main.bundle like postcss?

**Comment by shin-sforzando** [comment, marked as answer]
Since Congo uses Tailwind, the Tailwind JIT should remove unused styles within the Tailwind CSS.

For ALL unused styles, [PurgeCSS](https://purgecss.com/guides/hugo.html) is Hugo compatible, looks good.
